_search-plugins/star-tree-index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: default
+title: Star Tree index
+parent: Improving search performance
+nav_order: 54
+---
+
+# Star tree index
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress the feature or if you want to leave feedback, join the discussion on the [OpenSearch forum](https://forum.opensearch.org/).    
+{: .warning}
+
+Star Tree Index is a multi-field index that improves the performance of aggregations.
+
+OpenSearch will use the star-tree index to optimize aggregations based on the input query and star-tree configuration. No changes are required in the query syntax or requests.
+
+## Star tree index structure
+
+<img src="{{site.url}}{{site.baseurl}}/images/star-tree-index.png" alt="A Star Tree index containing two dimensions and two metrics" width="700">
+
+Star Tree index structure as portrayed in the above figure, consists of mainly two parts:
+- Star Tree and 
+- Sorted and aggregated star-tree documents backed by doc-values indexes
+    - The values are sorted based on the order of `ordered_dimension` , in the above example, first by `status` and then by `port` for each of the status.
+    - For each of the unique dimension values combinations , we also pre-compute the aggregated values for all the metrics such as `avg(size)` and `count(requests)` 
+
+Each node in the Star Tree points to a range of star-tree documents.
```

Comment:
Be consistent, either use Star Tree or star-tree, I see both getting used widely.

Reply:
Using Star Tree everywhere, thanks